File: CreatureCare/Models/Creature.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace CreatureCare.Models
{
    public class Creature
    {
        public int Id { get; set; }

        [Required]
        public string Name { get; set; }

        [Required]
        public string Type { get; set; }

        [Required]
        public string Origin { get; set; }

        [Required]
        public string Gender { get; set; }

        [Required]
        public string Birthdate { get; set; }

        public string ImageLocation { get; set; }

        public string Description { get; set; }

        public bool IsActive { get; set; }

        public int UserProfileId { get; set; }
        public UserProfile User { get; set; }
        public List<UserProfile> UserProfiles { get; set; }

        public Appointment Appointment { get; set; }
    }
}

File: CreatureCare/client/src/components/Creature.js
import React from "react";
import { Card } from "reactstrap";

const Creature = ({ creature }) => {
    return (
        <>

            <Card >
                <div>
                    <strong>{creature.imageLocation}</strong><br></br>
                </div>
                <div>
                    <strong>Name:</strong> {creature.name}<br></br>
                    <strong>Type:</strong> {creature.type}<br></br>
                    <strong>Origin:</strong> {creature.origin}<br></br>
                    <strong>Gender:</strong> {creature.gender}<br></br>
                    <strong>Birthdate:</strong> {creature.birthdate}<br></br>
                    <strong>Description:</strong> {creature.description}<br></br>
                </div>
                <div>
                    <strong>Owner:</strong> {creature.userProfile.firstName} {creature.userProfile.lastName}<br></br>
                    <strong>Email:</strong> {creature.userProfile.email}
                </div>
            </Card>

        </>
    );
};

export default Creature;



